pset-iterators.js

```diff
@@ -115,6 +135,14 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter((president, year1) => {
+  let officeTerm = president.took_office >= "1850" && president.left_office <= "1900"
+    return officeTerm
+  }).filter(function (officeTerm) {
+    return officeTerm.president
+  })
+
+console.log(presidentsBetween1850and1900)
```

Comment:
@jct117 Good job putting together this solution to find all of the presidents who served between 1850 and 1900. Currently, I see you have two filter method but for this example one filter is fine. when you run the second filter it ass a president who served outside of the years 1850 - 1900. Also since we are checking to see who served during this period of time you should replace `president.left_office <= "1900"` with `president.took_office <= "1900"`. This will let us know if any president took office between 1850 and 1900